<?xml version="1.0" encoding="UTF-8"?>
<!-- Vị trí: /ui/widgets/librarypaginationwidget.ui -->
<!-- Phiên bản: 1.0 (Mới) -->
<ui version="4.0">
 <class>LibraryPaginationWidget</class>
 <widget class="QWidget" name="LibraryPaginationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string>Hiển thị 0 - 0 trên tổng số 0 bản ghi</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="firstButton">
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
     <property name="toolTip">
      <string>Trang đầu</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="previousButton">
     <property name="text">
      <string>&lt;</string>
     </property>
     <property name="toolTip">
      <string>Trang trước</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="pageLineEdit">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="totalPagesLabel">
     <property name="text">
      <string>/ 1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="nextButton">
     <property name="text">
      <string>&gt;</string>
     </property>
     <property name="toolTip">
      <string>Trang sau</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="lastButton">
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
     <property name="toolTip">
      <string>Trang cuối</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
